python/desc/primes/Primes.py

```diff
@@ -6,6 +6,9 @@
 def is_prime(number):
     "Test if number is prime."
 
+    if number == 0 or number == 1:
```

Comment:
These changes look fine and address the reported bug, but I'm wondering if we should also check the input for negative numbers.  I'll open another issue on that.